tensorflow/lite/micro/examples/hello_world/evaluate_test.cc

```diff
@@ -24,6 +24,20 @@ limitations under the License.
 #include "tensorflow/lite/micro/system_setup.h"
 #include "tensorflow/lite/schema/schema_generated.h"
 
+namespace {
+using HelloWorldOpResolver = tflite::MicroMutableOpResolver<1>;
+alignas(alignof(HelloWorldOpResolver)) uint8_t
+    op_resolver_buffer[sizeof(HelloWorldOpResolver)];
+}  // namespace
+
+namespace tflite {
+TfLiteStatus GetOpResolver(HelloWorldOpResolver** op_resolver) {
+  *op_resolver = new (op_resolver_buffer) HelloWorldOpResolver();
+  TF_LITE_ENSURE_STATUS((*op_resolver)->AddFullyConnected());
+  return kTfLiteOk;
```

Comment:
there's not a very good reason to do a placement new here.

I would suggest something like:

```cc
// Registers the specific OPs needed for the current model.
TfLiteStatus RegisterOps(HelloWorldOpResolver& op_resolver) {
  TF_LITE_ENSURE_STATUS(op_resolver.AddFullyConnected());
  return kTfLiteOk;
}
```

Reply:
done